index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
```

Comment:
`signal` can also be an integer. That integer is OS-specific (`os.constants.signals.SIGTERM`).

Reply:
I added this possibility now: 
8f590cb48d61837c1ba4572f743a92decdfa4c01